src/App.tsx

```diff
@@ -1,109 +1,22 @@
 import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { PostList } from './components/PostList';
+
+const configuredPosts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId),
```

Comment:
will be better to set null if there is no user in usersFromServer array
in this way you can be sure that you definitely set null in this prop, and it's not undefined value because of you don't have this prop